common/FileUtils.java

```diff
@@ -265,9 +274,17 @@ public FileVisitResult preVisitDirectory(final Path dir, final BasicFileAttribut
           @Override
           public FileVisitResult visitFile(final Path file, final BasicFileAttributes attrs)
               throws IOException {
-            if (ignored != null) {
-              if (Pattern.matches(ignored, file.getFileName().toString())) {
-                return FileVisitResult.CONTINUE;
+            final String filePath = file.toString();
```

Comment:
Let's only ignore at the folder level. If it's a file - we always copy it.

Reply:
Done.